app/renderer/components/common/browserButton.js

```diff
@@ -88,7 +88,7 @@ const styles = StyleSheet.create({
     cursor: 'pointer',
     lineHeight: 1.25,
     width: 'auto',
-    height: 'auto',
+    height: '32px',
```

Comment:
Otherwise: simply removing the font-size and padding specification on button.less makes the buttons size standard.